service/test/CpuinfoIOGroupTest.cpp

```diff
@@ -32,6 +21,7 @@ extern "C"
 #include "geopm/PluginFactory.hpp"
 #include "geopm/PlatformTopo.hpp"
 #include "geopm_test.hpp"
+#include "MockCpuid.hpp"
```

Comment:
I think this newly-added header is unused. Did you plan to add the new cpuid class to this the CpuinfoIOGroup interface, or is the header left over from an earlier design?

Reply:
Left overs! Thanks